### PR TITLE
Refactor ngraph builders 6

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - Refactor some builders. Split headers per operation. Move to new directory

### Tickets:
 - [CVS-117139](https://jira.devtools.intel.com/browse/CVS-117139)

### CODE REVIEW

**Comment by olpipi** [comment]
Hi @vurusovs @iefode 
What you think about splitting builders.hpp header to set of small headers for every operation?
I can merge them into one big but **light weight** header. It will not affect building time like legacy builders.hpp does.

**Comment by vurusovs** [comment]
> Hi @vurusovs @iefode What you think about splitting builders.hpp header to set of small headers for every operation? I can merge them into one big but **light weight** header. It will not affect building time like legacy builders.hpp does.

Yes, let's do this

File: src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_bag_offsets_sum.hpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/test_enums.hpp"
```
Comment (vurusovs) [inline src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_bag_offsets_sum.hpp:5]: not used

Comment (olpipi) [reply src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_bag_offsets_sum.hpp:5]: done

**Review by vurusovs — APPROVED** [review]

**Pull request merged**